settings/LoanPolicyForm.js

```diff
@@ -114,6 +115,10 @@ class LoanPolicyForm extends React.Component {
     if (altRenewPeriod && altRenewPeriod.intervalId && !altRenewPeriod.duration) {
       this.props.change('renewalsPolicy.period.duration', 1);
     }
+    const openingTimeOffset = (allValues.loansPolicy && allValues.loansPolicy.openingTimeOffset);
+    if (openingTimeOffset && openingTimeOffset.intervalId && !openingTimeOffset.duration) {
+      this.props.change('loansPolicy.openingTimeOffset.duration', 0);
```

Comment:
Seems reasonable for me to use destructured "change" function here and above.  
```suggestion
     const { change } = this.props;
     
    ...
     
     change('loansPolicy.openingTimeOffset.duration', 0);
```